config/features.yml

```diff
@@ -340,6 +340,10 @@ features:
   mobile_api:
     actor_type: user
     description: API endpoints consumed by the VA Mobile App (iOS/Android)
+  mobile_appointment_requests:
```

Comment:
Not sure if I truly need to add this, but it makes it easier to flip the flag.

Reply:
I ended up later removing this because it was mysteriously breaking specs. But now I've re-added it and realize the spec failures are because flipper.rb turns all features on in the test environment. I didn't expect that behavior, but it's clearly written in the code.